src/routers/userControllers.js

```diff
@@ -0,0 +1,39 @@
+import express from 'express';
+import { createValidator } from 'express-joi-validation';
+import validationSchema from '../services/userValidation';
+import { createUser, updateUser, findAllUsers, deleteUser } from '../services/usersService';
+
+const router = express.Router();
+const validator = createValidator();
+
+router.route('/')
+    .get((req, res) => findAllUsers()
+        .then(users => res.send(users))
+        .catch(err => console.error(err))
+    )
+
+    .post(validator.body(validationSchema), (req, res) => {
+        createUser(req.body)
+            .then((us) => res.send(us))
+            .catch(err => console.error(err));
+    });
+
```

Comment:
Is there a possibility to request only desired users with `limit` and `loginSubstring` parameters?